int8 Class
float64 probability
int64 xmin
int64 ymin
int64 xmax
int64 ymax
float64 worldX
float64 worldY
float64 worldZ
